app/api/routes.py

```diff
@@ -244,13 +242,13 @@ def update_votes(id, vote_direction):
     except Exception as e:
         print_tb(e.__traceback__)
         print(e)
-        return standardize_response(None, None, None, 500)
+        return standardize_response(payload=None, status_code=500)
 
     initial_count = getattr(resource, vote_direction)
     setattr(resource, vote_direction, initial_count+1)
     db.session.commit()
 
-    return standardize_response(resource.serialize, None, "ok")
+    return standardize_response(payload=dict(data=resource.serialize), status_code=200)
```

Comment:
Since the default for `status_code` is 200, I don't think we need that to be specified here